build.py

```diff
@@ -208,99 +163,98 @@ def build_variable_fonts(designspace, *steps):
         )
 
     if args.static_fonts == True:
+        # Prepare masters for interpolation.
+        generator = fontmake.instantiator.Instantiator.from_designspace(designspace)
+
         for instance_descriptor in designspace.instances:
+            # Generate instances once.
+            instance = generator.generate_instance(instance_descriptor)
+            glyphsLib.interpolation.apply_instance_data_to_ufo(instance, instance_descriptor, designspace)
```

Comment:
I'm not sure I understand the purpose of using glyphsLib here.

Reply:
It has a function `apply_instance_data_to_ufo` that lets you use various custom parameters Glyphs knows and will apply them to your UFO, e.g. setting PANOSE. Most of them are not known to the Designspace spec, so this is a way to apply them to instances at least.

Reply:
Got it :).